controllers/managedocs_controller.go

```diff
@@ -925,26 +1008,28 @@ func (r *ManagedOCSReconciler) reconcilePrometheusService() error { //nolint:dea
 
 		r.prometheusService.Spec.Ports = []corev1.ServicePort{
 			{
-				Name:       "odf-prometheus-endpoint",
+				Name:       "https",
 				Protocol:   corev1.ProtocolTCP,
 				Port:       9339,
-				TargetPort: intstr.FromString("web"),
+				TargetPort: intstr.FromString("https"),
 			},
 		}
 		r.prometheusService.Spec.Selector = map[string]string{
-			monLabelKey: monLabelValue,
+			prometheusServiceSelectorLabelKey: prometheusServiceSelectorLabelValue,
 		}
 		utils.AddAnnotation(r.prometheusService,
 			servingCertSecretBetaAnnotationKey, prometheusServiceSecretName)
 		utils.AddAnnotation(r.prometheusService,
 			servingCertSecretAlphaAnnotationKey, prometheusServiceSecretName)
+		utils.AddLabel(r.prometheusService,
+			prometheusServiceLabelKey, prometheusServiceLabelValue)
```

Comment:
No need to invent a new set of labels, please use our existing `monLabelKey`, `monLabelValue` 

```suggestion
		utils.AddLabel(r.prometheusService, monLabelKey, monLabelValue)
```

Reply:
We need that label on the service to use the Managed-tenats federation mechanism

Reply:
Are they targeting that specific label? Isn't that configurable in the addon manifest?

Reply:
Yes, they are targeting that label and it is not configurable.

Reply:
In that case there is no need for constants just add the label inline with a comment above explaining the need for this label on the service